src/core/components/progress/progress.stories.tsx

```diff
@@ -26,6 +26,19 @@ export const Default: Story = {
         return <Progress {...props} />;
     },
 };
+
+/**
+ *  Progress component with indicator example.
+ */
+export const WithIndicator: Story = {
+    args: {
+        value: 70,
+        indicator: 50,
+    },
+    render: (props) => {
+        return <Progress {...props} />;
+    },
+};
```

Comment:
The `render` function here is probably redundant, we can just specify the `args`. Same for the `Default` story